src/screens/HomePage/HomePage.js

```diff
@@ -14,7 +15,26 @@ import {
   PartnerHeader,
 } from './HomePage.style';
 
+// TODO: 180 is rough accounting for the navbar and searchbar;
+// could make these constants so we don't have to manually update
```

Comment:
Seems like we don't really use this 180 default (we explicitly pass in -80), so maybe remove the default and then you don't need this TODO? 😁 

Else, change the default to -80 if that makes more sense... not sure what the -80 represents.